cms/models.py

```diff
@@ -77,7 +124,12 @@ class BootcampRunPage(BootcampPage):
         help_text="The bootcamp run for this page",
     )
 
-    content_panels = BootcampPage.content_panels + [FieldPanel("bootcamp_run")]
+    content_panels = [FieldPanel("bootcamp_run")] + BootcampPage.content_panels
+
+    @property
+    def product(self):
+        """Gets the product associated with this page"""
+        return self.bootcamp_run
```

Comment:
There is no reason for this, please remove. We already have `bootcamp_run` and are not looking to add multiple types of products right now. This looks like a wasteful pre-optimization.

Reply:
@ahmed-belal I am using this for slug.

Reply:
Please use `bootcamp_run`, there is no need to create a duplicate property just for a single check.

Reply:
my bad, I am using the `self.bootcamp_run.run_key` below but forgot to remove this property. It was not in use.